Box.V2.Core/Box.V2.Core.csproj

```diff
@@ -1,24 +1,18 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <Description>Box.V2 for .Net Core</Description>
-    <AssemblyTitle>Box.V2.Core</AssemblyTitle>
-    <Version>5.8.0</Version>
-    <Authors>Box, Inc.</Authors>
-	<Owners>Box, Inc.</Owners>
-	<PackageProjectUrl>https://github.com/box/box-windows-sdk-v2</PackageProjectUrl>
-  <PackageLicenseExpression>Apache-2.0</PackageLicenseExpression>
-	<PackageRequireLicenseAcceptance>false</PackageRequireLicenseAcceptance>
-	<PackageDescription>Windows SDK for v2 of the Box API. The SDK is built for .NET Core and includes JWTAuth support.</PackageDescription>
-	<PackageReleaseNotes>See https://github.com/box/box-windows-sdk-v2/blob/main/CHANGELOG.md#580-2024-07-22</PackageReleaseNotes>
-	<Copyright>Copyright 2021</Copyright>
-	<PackageTags>Box;V2;SDK;Platform;Enterprise;Collaboration;Storage;File Management</PackageTags>
+    <PackageId>Box.V2.Core</PackageId>
     <TargetFramework>netstandard2.0</TargetFramework>
+    <Version>5.8.0</Version>
+    <Description>Box SDK for v2 of the Box API. The SDK is targeting .Net Standard 2.0</Description>
+    <Authors>Box Inc.</Authors>
+    <AssemblyTitle>Box.V2.Core</AssemblyTitle>
+	  <PackageProjectUrl>https://github.com/box/box-windows-sdk-v2</PackageProjectUrl>
```

Comment:
In Box.V2.csproj there is also a `RepositoryUrl` tag `<RepositoryUrl>https://github.com/box/box-windows-sdk-v2.git</RepositoryUrl>`. Shouldn't we add it here as well?

Reply:
Good catch! I'll add it

Reply:
Hmm, looks like it's a SourceLink property and it is used to set `RepositoryUrl` anyway https://github.com/dotnet/sourcelink/issues/659#issuecomment-711288537. So I don't think we need to add it here.